.github/workflows/wkdev-sdk.yml

```diff
@@ -10,14 +10,12 @@ defaults:
   run:
     shell: bash
 jobs:
-  build:
-    runs-on: self-hosted
+  buildintel:
+    runs-on: [self-hosted, x64]
```

Comment:
Why should it run on both self-hosted==amd64, and x64 (I guess that's the name of your aarch64 runner?). Both names for the runners are bad, I'd prefer something like linux-amd64, linux-aarch64, etc.